jedi.inc

```diff
@@ -1214,6 +1217,213 @@
 
 {$ENDIF BORLAND}
 
+{------------------------------------------------------------------------------}
+{ Oxygene support                                                              }
+{ Since largely a superset of Delphi, define macros expecting Delphi           }
+{------------------------------------------------------------------------------}
+{$IFDEF ELEMENTS}
+  // CLR and NETSTANDARD or NETCORE are auto-defined
+  // LINUX is defined for both Linux AND Android
+  // https://docs.elementscompiler.com/Compiler/ConditionalDefines/
+
+  {$DEFINE PUREPASCAL}
+
```

Comment:
This really should be placed above the `$IFDEF BORLAND` blocks so that it is merged with the `$IFNDEF` you added.

Reply:
I believe FPC was an "afterthought" and so it was deemed easier to "undef" `BORLAND` in FPCs section.

That's why I may also go the same as FPC: 
- Leave $DEFINE BORLAND alone
- UNDEF it inside an $IFDEF ELEMENTS that is placed after the $IFDEF FPC section.

```
${ENDIF FPC}

{$IFDEF ELEMENTS}
  // Elements doesn't support CPU assembly code.
  {$DEFINE PUREPASCAL}
  {$UNDEF BORLAND}
{$ENDIF ELEMENTS}
```

The comment is a must have for me, but for the rest, that's just a thought, I'm open to suggestions. @ronaldhoek , what do you think?